documentation/modules/vmware-privileges.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="vmware-privileges_{context}"]
+= VMware privileges
+
+The following VMware privileges are required to migrate virtual machines to {virt} with the {project-first}.
+
+[cols="2", options="header"]
+.VMware privileges
+|===
+|Privilege |Description
+|`Virtual machine.Interaction.Power Off`   |Allows powering off a powered-on virtual machine. This operation powers down the guest operating system.
+|`Virtual machine.Interaction.Power On`  |Allows powering on a powered-off virtual machine and resuming a suspended virtual machine.
+|`Virtual machine.Provisioning.Allow disk access`   |Allows opening a disk on a virtual machine for random read and write access. Used mostly for remote disk mounting.
+|`Virtual machine.Provisioning.Allow read-only disk access`   |Allows opening a disk on a virtual machine for random read access. Used mostly for remote disk mounting.
```

Comment:
Please see in the doc bug description, the VMware user should have all Virtual machine.Provisioning privileges

Reply:
@mguetta1  What  other privileges are there in Virtual Machine Provisioning besides 'Allow disk access' and 'Allow read-only disk access'?

Reply:
@sbeskin-redhat could be that it depends on the VMware version. Attached is the list for Provisioning privileges in VMware 6.5
![image](https://user-images.githubusercontent.com/60384172/152786634-e8ba5285-f7fb-4522-80eb-96ee510d613e.png)

Reply:
Where did you get this list? Are there any descriptions of what these privileges allow?